syf_actor_example.json

```diff
@@ -0,0 +1 @@
ne": false}, {"now_state": [[0.07335574045407602, -0.032404541205004994, -0.028813271891375544, -0.0807737758194862]], "action": 0, "reward": 1.0, "next_state": [[0.07270764962997592, -0.22710185604503605, -0.03042874740776527, 0.20268104600715858]], "done": false}, {"now_state": [[0.07270764962997592, -0.22710185604503605, -0.03042874740776527, 0.20268104600715858]], "action": 0, "reward": 1.0, "next_state": [[0.0681656125090752, -0.42177571817503223, -0.026375126487622098, 0.4856119907063964]], "done": false}, {"now_state": [[0.0681656125090752, -0.42177571817503223, -0.026375126487622098, 0.4856119907063964]], "action": 0, "reward": 1.0, "next_state": [[0.059730098145574555, -0.6165157440645279, -0.016662886673494172, 0.7698670444738287]], "done": false}, {"now_state": [[0.059730098145574555, -0.6165157440645279, -0.016662886673494172, 0.7698670444738287]], "action": 0, "reward": 1.0, "next_state": [[0.047399783264284, -0.8114044450359776, -0.0012655457840175972, 1.0572608514845399]], "done": false}, {"now_state": [[0.047399783264284, -0.8114044450359776, -0.0012655457840175972, 1.0572608514845399]], "action": 1, "reward": 1.0, "next_state": [[0.031171694363564446, -0.6162657470499474, 0.0198796712456732, 0.7641809685457819]], "done": false}, {"now_state": [[0.031171694363564446, -0.6162657470499474, 0.0198796712456732, 0.7641809685457819]], "action": 0, "reward": 1.0, "next_state": [[0.018846379422565496, -0.8116557520703436, 0.035163290616588835, 1.0630523025781153]], "done": false}, {"now_state": [[0.018846379422565496, -0.8116557520703436, 0.035163290616588835, 1.0630523025781153]], "action": 0, "reward": 1.0, "next_state": [[0.002613264381158626, -1.0072251342484915, 0.056424336668151144, 1.3665609118378539]], "done": false}, {"now_state": [[0.002613264381158626, -1.0072251342484915, 0.056424336668151144, 1.3665609118378539]], "action": 0, "reward": 1.0, "next_state": [[-0.017531238303811203, -1.203006269739366, 0.08375555490490821, 1.6763452110360322]], "done": false}, {"now_state": [[-0.017531238303811203, -1.203006269739366, 0.08375555490490821, 1.6763452110360322]], "action": 0, "reward": 1.0, "next_state": [[-0.04159136369859852, -1.3989940692674632, 0.11728245912562886, 1.9938917267472402]], "done": false}, {"now_state": [[-0.04159136369859852, -1.3989940692674632, 0.11728245912562886, 1.9938917267472402]], "action": 1, "reward": 1.0, "next_state": [[-0.06957124508394778, -1.2052789616213269, 0.15716029366057366, 1.7397172630580449]], "done": false}, {"now_state": [[-0.06957124508394778, -1.2052789616213269, 0.15716029366057366, 1.7397172630580449]], "action": 0, "reward": 1.0, "next_state": [[-0.09367682431637432, -1.4018034782938988, 0.19195463892173456, 2.0768861580207334]], "done": false}, {"now_state": [[-0.09367682431637432, -1.4018034782938988, 0.19195463892173456, 2.0768861580207334]], "action": 1, "reward": -10, "next_state": [[-0.1217128938822523, -1.209077724684461, 0.23349236208214924, 1.849195889235618]], "done": true}]
```

Comment:
It is very bad to upload your data files. Is it related to the sources? Try [.gitignore](https://docs.github.com/en/free-pro-team@latest/github/using-git/ignoring-files).